### PR TITLE
bots: Drop rhel-7-5 image

### PR DESCRIPTION
**Pull request by martinpitt**
Now that RHEL 7.6 is released, this is obsolete.

This gets rid of a lot of old special cases in our tests.

Also flip rhel-atomic to build on rhel-7-6. The most recent rhel-atomic
image refresh already caught a 7.6 based image.

### CODE REVIEW

**Comment by martinpitt** [comment]
Force-pushed trivial pyflakes error (now obsolete "subprocess" import)

**Comment by martinpitt** [comment]
7-5-distropkg is expected to fail here, as this PR drops the image and this test. Everything else succeeded, aside from the rhel-7-6/welder tests which are a known unrelated issue.

File: bots/image-refresh [hunk 1]
```diff
@@ -62,23 +62,19 @@ TRIGGERS = {
...
+        "verify/rhel-7-6"
     ],
     "ipa": [
         "verify/fedora-28",
-        "verify/rhel-7-5",
-        "verify/rhel-7-5-distropkg",
-        "verify/ubuntu-1604",
+        "verify/rhel-7-6",
+        "verify/rhel-7-6-distropkg",
+        "verify/ubuntu-1804",
```
Comment (larskarlitski) [inline bots/image-refresh]: Isn't this ubuntu change unrelated to this patch?

Comment (martinpitt) [reply bots/image-refresh]: Ah sorry, it is; probably should have been a separate commit. ubuntu-1604 doesn't even exist any more.

OK to mention this in  the commit message when merging, or want me to split it out?

Comment (martinpitt) [reply bots/image-refresh]: Split out into separate commit, thanks for catching. Test results copied with push-rewrite.

**Pull request merged**